app/http.php

```diff
@@ -0,0 +1,346 @@
+        $route = $utopia->match($request);
+        logError($error, "httpError", $logger, $route);
+
+        switch ($error->getCode()) {
+            case 400: // Error allowed publicly
+            case 401: // Error allowed publicly
+            case 402: // Error allowed publicly
+            case 403: // Error allowed publicly
+            case 404: // Error allowed publicly
+            case 406: // Error allowed publicly
+            case 409: // Error allowed publicly
+            case 412: // Error allowed publicly
+            case 425: // Error allowed publicly
+            case 429: // Error allowed publicly
+            case 501: // Error allowed publicly
+            case 503: // Error allowed publicly
+                $code = $error->getCode();
+                break;
+            default:
+                $code = 500; // All other errors get the generic 500 server error status code
+        }
+
+        $output = [
+            'message' => $error->getMessage(),
+            'code' => $error->getCode(),
+            'file' => $error->getFile(),
+            'line' => $error->getLine(),
+            'trace' => $error->getTrace(),
+            'version' => App::getEnv('OPEN_RUNTIMES_PROXY_VERSION', 'UNKNOWN')
+        ];
+
+        $response
+            ->addHeader('Cache-Control', 'no-cache, no-store, must-revalidate')
+            ->addHeader('Expires', '0')
+            ->addHeader('Pragma', 'no-cache')
+            ->setStatusCode(\intval($code));
+
+        $response->json($output);
+    });
+
+/** @phpstan-ignore-next-line */
+Co\run(
+    function () use ($register) {
+        /** @phpstan-ignore-next-line */
+        \Swoole\Coroutine\batch([
+                function () use ($register) {
+                    // If no health check, mark all as online
+                    if (App::getEnv('OPEN_RUNTIMES_PROXY_HEALTHCHECK', 'enabled') === 'disabled') {
+                        /**
+                         * @var Table $state
+                         */
+                        $state = $register->get('state');
+                        $executors = \explode(',', (string) App::getEnv('OPEN_RUNTIMES_PROXY_EXECUTORS', ''));
+
+                        foreach ($executors as $executor) {
+                            $state->set($executor, [
+                                'status' => 'online',
+                                'hostname' => $executor,
+                                'state' =>  \json_encode([])
+                            ]);
+                        }
+
+                        return;
+                    }
+
+                    // Initial health cehck + start timer
+                    healthCheck($register, true);
+
+                    $defaultInterval = '10000'; // 10 seconds
+                    Timer::tick(\intval(App::getEnv('OPEN_RUNTIMES_PROXY_HEALTHCHECK_INTERVAL', $defaultInterval)), fn () => \go(fn () => healthCheck($register, false)));
```

Comment:
Here as well. Im really unsure if `go()` is doing what its expected to do.


the ideal usage of go would be something like this 

for loop {
   go ( // Do something ) 
}

this is how true concurrency is achieved with go.

In our usage I see go running a single function like so -> `go (fn)` which just invokes go once thereby not achieving full concurrency

Reply:
Removed go method.